src/components/Announce/index.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import RaisedButton from 'material-ui/RaisedButton';
+import strings from 'lang';
+import { connect } from 'react-redux';
+import { getGithubPulls } from 'actions/githubPulls';
+import ReactMarkdown from 'react-markdown';
+
+import styles from './styles.css';
+
+const firstPrio = 'blog';
```

Comment:
instead of priorities I think we should have two elements, one for releases and one for blog.  If there is both a recent release and a blog we want to show both?